dev/docker-compose.yaml

```diff
@@ -75,6 +78,24 @@ services:
     profiles:
       - fe
 
+  test_be_next:
```

Comment:
would it make sense to take an approach similar to the mongo seeding? basically, we create a `be_next_base` with `profile: never` that contains the config shared between `be_next` and `test_be_next`. Then these 2 services `extends` that one and in the `test_be_next` we mount the additional `.test.env` file, making sure to only specify the ENVs that are not already set by the `.env` file